### ISSUE TITLE
Normalize exercices numbering format

### ISSUE THREAD

**Issue by helenemccarron** [Connexions/cnx-recipes#617]
All books  end of chapter exercises and end of book solutions should use the following format:

#. Exercise

Example: 1. Exercise description....

```
<span class="os-number">1</span>
<span class="os-divider">. </span>
<div class="os-problem-container">Exercise description .... </div>
```

This should prevent future issues with spacing between the number, the divider and the content in exercises.

**Comment by helenemccarron** [comment]
This was resolved in PR #613 (merged into pr #615)

**Comment by wilkus27** [comment]
Verified on katalyst01, e.g.

**College Physics** (https://katalyst01.cnx.org/contents/Ax2o07Ul@11.33:aDS9F9Oz@11.33/Conceptual-Questions)

![physics](https://user-images.githubusercontent.com/40228854/45437478-2b80d780-b6b5-11e8-9909-2decf56d14e5.JPG)

**Concepts of Biology** (https://katalyst01.cnx.org/contents/s8Hh0oOc@11.15:_uFXixPr@11.15/Review-Questions)

![biology](https://user-images.githubusercontent.com/40228854/45437602-6aaf2880-b6b5-11e8-8dba-887f21d2022b.JPG)
